app/src/main/java/com/randomappsinc/studentpicker/home/NameListsAdapter.java

```diff
@@ -146,20 +160,19 @@ public void renameList() {
         public void deleteList() {
             showDeleteDialog(position);
         }
-    }
 
-    @Override
-    public View getView(int position, View view, ViewGroup parent) {
-        NameListViewHolder holder;
-        if (view == null) {
-            LayoutInflater vi = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
-            view = vi.inflate(R.layout.names_list_item, parent, false);
-            holder = new NameListViewHolder(view);
-            view.setTag(holder);
-        } else {
-            holder = (NameListViewHolder) view.getTag();
+        @Override
+        public void onClick(View v) {
+            if (listItemClickListener != null) {
```

Comment:
This should never be null; let's not check against that. Nesting makes the code less readable, so we have nothing to gain here and something to lose.